### ISSUE TITLE
Allow user's to edit their username

### ISSUE THREAD

**Issue by benrmatthews** [goodhere/goodhere#18]
We need user's to be able to provide a username in order to show their profile. We should store this in the users table. We should also make sure to update the Auth0 nickname for consistency.

**Comment by Sourav59580** [comment]
can you elaborate more?

**Comment by benrmatthews** [comment]
Hi @Sourav59580 - we’ve been using this tutorial to implement user authentication using Auth0:

https://auth0.com/blog/complete-guide-to-react-user-authentication/

We have completed the integration with Auth0, including sign in and log out. Click the “sign in” button in the nav bar of https://goodhere.org to see it in action.

What we need to implement is:

* User profile page
* Ability for users to edit their own profile

I feel that this issue could be broken down more or there are a few steps to implement first before username editing is added.

Please add any comments to how we could best approach this issue.

**Comment by Sourav59580** [comment]
sir, my approach is, after login users can go their own profile section route where he can edit everything like username, full details everything.

**Comment by benrmatthews** [comment]
@Sourav59580 Yes, that sounds great. Do you need anything else to begin work on this?

**Comment by Sourav59580** [comment]
My best approach is create "profile icon or username" in the navbar instead of “Sign out” button in the nav bar of https://goodhere.org. After clicking the "profile icon" one dropdown will be open and where you will give profile, setting,  sign out dropdown item.
https://linkofy.files.wordpress.com/2012/04/example.png

**Comment by benrmatthews** [comment]
@Sourav59580 That sounds great!
